dashboard/src/components/AppView/AppView.test.tsx

```diff
@@ -114,4 +122,50 @@ describe("AppViewComponent", () => {
       expect(sockets.length).toEqual(0);
     });
   });
+
+  describe("renderization", () => {
+    it("renders all the elements of an application", () => {
+      const wrapper = shallow(<AppViewComponent {...validProps} />);
+      expect(wrapper.find(ChartInfo).exists()).toBe(true);
+      expect(wrapper.find(DeploymentStatus).exists()).toBe(true);
+      expect(wrapper.find(AppControls).exists()).toBe(true);
+      expect(wrapper.find(ServiceTable).exists()).toBe(true);
+      expect(wrapper.find(AppNotes).exists()).toBe(true);
+      expect(wrapper.find(AppDetails).exists()).toBe(true);
+    });
+
+    it("renders an error if it exists", () => {
```

Comment:
I read this as "if the app exists" which obviously in this case the app doesn't exist, but I think you mean that the `error` exists. Maybe we can reword this to say "renders an error if error prop is set" to make it very clear?